apps/desktop-wallet/src/components/SnackbarManager.tsx

```diff
@@ -90,6 +91,7 @@ const SnackbarPopup = styled(motion.div)`
   backdrop-filter: blur(10px);
   max-width: 800px;
   word-wrap: break-word;
+  overflow: scroll;
```

Comment:
The above 2 changes make sure that when a very long error gets displayed, the height of the snackbar doesn't grow too much and that the content can be scrolled. Imporant that the content is scrolled to top so that when the user takes a screenshot, we can see the top of the error message.

https://github.com/alephium/alephium-frontend/assets/1579899/874e714b-6565-4c6d-b850-4befeb165d99